www/src/components/Header.tsx

```diff
@@ -28,11 +28,13 @@ export const Header: FC<HeaderProps> = (props: HeaderProps) => {
                         {title}
                     </Link>
                 </div>
-                {sections.map((section) => (
-                    <Link color='textSecondary' key={section.title} variant='body2' href={section.url} className={classes.toolbarLink}>
-                        {section.title}
-                    </Link>
-                ))}
+                <Hidden xsDown>
+                    {sections.map((section) => (
+                        <Link color='textSecondary' key={section.title} variant='body2' href={section.url} className={classes.toolbarLink}>
+                            {section.title}
+                        </Link>
```

Comment:
Nit: since these are now just internal links could you add `component={RouterLink}` where `RouterLink` is imported from `react-router-dom` as 
```ts
import { Link as RouterLink } from 'react-router-dom';
```
You might also need to change `href` to `to`
For more details refer to [Header.tsx](https://github.com/UAlbertaCompEClub/compe-plus/blob/master/www/src/components/Header.tsx)
This will allow users to navigate faster between pages

You don't need to change the rest of the links since they're external